modules/tinymce/src/core/main/ts/focus/FocusController.ts

```diff
@@ -54,6 +58,11 @@ const getActiveElement = (editor: Editor): Element => {
   }
 };
 
+const shouldTrapFocusInFullScreen = (editor: Editor): boolean =>
+  editor.plugins.fullscreen
+    && editor.plugins.fullscreen.isFullscreen()
```

Comment:
This is specifically what I asked you not to do, we can't have core assuming things about plugin capabilities 😛